docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -9,7 +9,8 @@ videoBanner: GA37qqB6Lmk
 </Admonition>
 
 Passwordless takes advantage of WebAuthn to provide passwordless and
-usernameless authentication for Teleport.
+usernameless authentication for Teleport.  Teleport [local users](../../setup/admin/users.mdx) are the only initial 
```

Comment:
Something about the verbiage here doesn't sound quite right to me.

How about `Passwordless authentication is support for local users only` instead?

Reply:
I'm not sure the introductory paragraph of this guide is the best place to put this caveat. In this PR, half of the intro is a statement about what passwordless is and half is a specific warning about usage. In an introduction, I would expect a general explanation of what the feature is why you would use it. 

Maybe add a note like this to the Prerequisites instead:

```markdown
- At least one local Teleport user. Passwordless currently
only supports authentication for local users, and does 
not support single sign-on connectors.
```